cmd/slsa-github-generator/attest.go

```diff
@@ -130,7 +130,10 @@ run in the context of a Github Actions workflow.`,
 
 			if attPath != "" {
 				s := sigstore.NewDefaultSigner()
-				att, err := s.Sign(ctx, p)
+				att, err := s.Sign(ctx, &intoto.Statement{
```

Comment:
I think if we want to use cosign's `attest` API, we still need a way to pass a predicate only. Probably fine to leave as-is for now, and wait till we actually need it. (I use cosign attest via the CLI for the ko containers atm)
@asraa any thought?

Reply:
I don't think there's any way to write `AttestCmd` with outputs for the attestation or log entry right now -- (btw are you holding on to the uploaded log entry anywhere?)

I think we will need to wait for a refactor of that command to support Blobs, or a new `AttestBlobCmd`.

For the blob case, how would `Attest` compute the subject digest without either the binary blob as a second argument or the entire Statement?